nhais-adaptor/edifact/incoming/models/interchange.py

```diff
@@ -0,0 +1,28 @@
+class InterchangeHeader(object):

```

Comment:
Do these classes need to specify `object` as a base class?

Reply:
Probably not. Will remove 👍